core/build.gradle.kts

```diff
@@ -0,0 +1,81 @@
+@file:Suppress("SpellCheckingInspection", "UnstableApiUsage")
+
+// TODO: Remove this after https://youtrack.jetbrains.com/issue/KTIJ-19369 is resolved.
+@Suppress("DSL_SCOPE_VIOLATION")
+plugins {
+    alias(libs.plugins.android.library)
+    alias(libs.plugins.kotlin)
+}
+
+android {
+    compileSdk = 33
+
+    defaultConfig {
+        minSdk = 26
+        targetSdk = 33
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            isMinifyEnabled = false
+            proguardFiles(getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro")
+        }
+    }
+
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+
+    kotlinOptions {
+        jvmTarget = "1.8"
+    }
+
+    buildFeatures {
+        compose = true
+    }
+
+    composeOptions {
+        kotlinCompilerExtensionVersion = libs.versions.composeCompiler.get()
+    }
+
+    namespace = "com.imashnake.animite.core"
+
+    // Workaround for KSP generated sources not being indexable by the IDE
+    libraryVariants.all {
+        kotlin.sourceSets {
+            getByName(name) {
+                kotlin.srcDir("build/generated/ksp/$name/kotlin")
+            }
+        }
+    }
+}
+
+dependencies {
+    // AndroidX
+    implementation(libs.androidx.activityCompose)
+    implementation(libs.androidx.coreKtx)
+    implementation(libs.androidx.lifecycleRuntimeKtx)
+
+    // Compose
+    implementation(libs.compose.animation)
+    implementation(libs.compose.foundation)
+    implementation(libs.compose.material)
+    implementation(libs.compose.material3)
+    implementation(libs.compose.runtime)
+    implementation(libs.compose.ui)
+    debugImplementation(libs.compose.ui.tooling)
+    implementation(libs.compose.ui.toolingPreview)
+
+    // Kotlin
+    implementation(libs.kotlin.coroutines.android)
+    implementation(libs.kotlin.coroutines.core)
+
+    testImplementation(libs.test.junit)
+
+    androidTestImplementation(libs.androidx.test.junit)
```

Comment:
we're gonna actually use the dependencies one day, yeyeye?

Reply:
I hope so
cc @imashnake0, write tests :dagger:

Reply:
No :i_am_weeb: